goal_src/jak1/levels/rolling/rolling-robber.gc

```diff
@@ -59,6 +59,8 @@
   )
 
 (defbehavior fuel-cell-init-as-spline-slider fuel-cell ((arg0 handle) (arg1 float) (arg2 float) (arg3 int))
+
+     (when  (not (task-closed? (game-task rolling-robbers)(task-status need-resolution)))
```

Comment:
do you even need this with the check inside the rolling-robbers code? otherwise seems fine

Reply:
I don't but it didnt hurt to leave it, I can remove if you want.

Reply:
It was left in in my testing cycle

Reply:
meh I see its only used in rolling-robber code, should be okay